packaging/QueueNado.spec

```diff
@@ -40,9 +40,9 @@ else
 fi
 
 make -j6
-sudo ./UnitTestRunner
+#sudo ./UnitTestRunner
```

Comment:
that's OK. I will still review and comment and the comments will just go away once corrected. If nothing else it will help as a remainder